modin/pandas/general.py

```diff
@@ -51,44 +51,85 @@ def notna(obj):
+        These must be found in both DataFrames. If on is None and not merging on indexes
+        then this defaults to the intersection of the columns in both DataFrames.
+    left_on : label or list, or array-like
+        Column or index level names to join on in the left DataFrame.
+        Can also be an array or list of arrays of the length of the left DataFrame.
+        These arrays are treated as if they are columns.
+    right_on : label or list, or array-like
+        Column or index level names to join on in the right DataFrame.
+        Can also be an array or list of arrays of the length of the right DataFrame.
+        These arrays are treated as if they are columns.
+    left_index : bool, default False
+        Use the index from the left DataFrame as the join key(s).
+        If it is a MultiIndex, the number of keys in the other DataFrame
+        (either the index or a number of columns) must match the number of levels.
+    right_index : bool, default False
+        Use the index from the right DataFrame as the join key. Same caveats as left_index.
+    sort : bool, default False
+        Sort the join keys lexicographically in the result DataFrame.
+        If False, the order of the join keys depends on the join type (how keyword).
+    suffixes : tuple of (str, str), default ('_x', '_y')
+        Suffix to apply to overlapping column names in the left and right side, respectively.
+        To raise an exception on overlapping columns use (False, False).
+    copy : bool, default True
+        If False, avoid copy if possible.
+    indicator : bool or str, default False
+        If True, adds a column to output DataFrame called "_merge" with information
+        on the source of each row. If string, column with information on source of each row
+        will be added to output DataFrame, and column will be named value of string.
+        Information column is Categorical-type and takes on a value of "left_only"
+        for observations whose merge key only appears in 'left' DataFrame,
+        "right_only" for observations whose merge key only appears in 'right' DataFrame,
+        and "both" if the observation’s merge key is found in both.
+    validate : str, optional
+        If specified, checks if merge is of specified type.
+        - 'one_to_one' or '1:1': check if merge keys are unique in both left and right datasets.
+        - 'one_to_many' or '1:m': check if merge keys are unique in left dataset.
+        - 'many_to_one' or 'm:1': check if merge keys are unique in right dataset.
+        - many_to_many' or 'm:m': allowed, but does not result in checks.
+    Returns
+    -------
+    DataFrame
+        A DataFrame of the two merged objects.
+    """
     if not isinstance(left, DataFrame):
         raise ValueError(
-            "can not merge DataFrame with instance of type {}".format(type(right))
+            "The left object has to be DataFrame whereas it is {}".format(type(left))
```

Comment:
Is this the same error message pandas provides?

Reply:
I checked pandas error messages and aligned them with our.